nixos/maintainers/scripts/hcloud/create-image.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env nix-shell
+#! nix-shell -i bash -p hcloud-upload-image
+
+set -euo pipefail
+
+if [[ $# -ne 1 ]]; then
+    echo >&2 "Usage: $0 <x86|arm>"
+    exit 64
+fi
+
+hetzner_arch="$1"
+case "$hetzner_arch" in
+    x86)
+        nix_arch="x86_64"
+        ;;
+    arm)
+        nix_arch="aarch64"
+        ;;
+    *)
+        echo >&2 "Unknown Hetzner architecture: ${hetzner_arch}"
+        exit 64
+        ;;
+esac
+
+nix-build '<nixpkgs/nixos/lib/eval-config.nix>' \
```

Comment:
We should use a relative path here so that it works without channels

Reply:
Oops, I misinterpreted this comment as: relative path in https://github.com/NixOS/nixpkgs/blob/ad666bb4ea71d820c96bb7fcd511f1ba14bf90fa/nixos/modules/virtualisation/hcloud-image.nix#L26.

Do you think a relative path is better there as well, or should I revert that? Seems the configs in virtualisation are mixed on that aspect.

Reply:
Note that, for `create-image.sh`, the shebang still relies on `NIX_PATH` for `hcloud-upload-image`.